packages/atomic/src/components/ipx/atomic-ipx-button/atomic-ipx-button.tsx

```diff
@@ -19,6 +19,11 @@ export class AtomicIPXButton {
    */
   @Prop({reflect: true}) public label = 'Workplace Search';
 
+  /**
+   * The button icon.
+   */
+  @Prop({reflect: true}) public icon = '';
```

Comment:
```suggestion
  @Prop({reflect: true}) public icon?: string;
```
Then the check on line 46 can just be `this.icon ?`